uaclient/config.py

```diff
@@ -577,9 +441,9 @@ def delete_cache_key(self, key: str) -> None:
             raise RuntimeError(
                 "Invalid or empty key provided to delete_cache_key"
             )
-        if key.startswith("machine-access") or key == "machine-token":
-            self._entitlements = None
-            self._machine_token = None
+        if key.startswith("machine-access"):
+            # will find a better way for this
```

Comment:
is this comment still valid?

Reply:
if so, can you expand on what is wrong about the current way?

Reply:
It is not valid currently. I put this comment before the new file object.